docker/db/mysql/init/init.sql

```diff
@@ -0,0 +1,26 @@
+GRANT ALL PRIVILEGES ON *.* TO 'root'@'localhost';
+FLUSH PRIVILEGES;
+
+CREATE DATABASE IF NOT EXISTS `chess` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
+CREATE DATABASE IF NOT EXISTS `chess-test` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
+
+USE chess;
+
+CREATE TABLE IF NOT EXISTS game_information
+(
+    game_id            int        NOT NULL AUTO_INCREMENT,
+    current_turn_color VARCHAR(5) NOT NULL,
+    PRIMARY KEY (game_id)
+);
+
+CREATE TABLE IF NOT EXISTS chessboard
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS chess_boards
```
테이블 명은 보통 복수로 만드는게 일반적일 것 같네요